packages/react-components/react-virtualizer/library/src/components/Virtualizer/Virtualizer.types.ts

```diff
@@ -171,6 +189,11 @@ export type VirtualizerConfigProps = {
    * Imperative ref contains our scrollTo index functionality for user control.
    */
   imperativeVirtualizerRef?: RefObject<VirtualizerDataRef>;
+
+  /**
+   * A ref that provides the size of container (vertical - height, horizontal - width), set by a resize observer.
```

Comment:
Might also be helpful to mention `useStaticVirtualizerMeasure` can provide this for free.